src/NetAppFiles/NetAppFiles.Test/ScenarioTests/AccountTests.ps1

```diff
@@ -0,0 +1,107 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+function Test-AccountCrud
+{
+    $resourceGroup = "pws-sdk-tests-rg-1"
+    $accName1 = "pws-sdk-acc-1"
+    $accName2 = "pws-sdk-acc-2"
+    $accName3 = "pws-sdk-acc-3"
+    $resourceLocation = "westus2"
+
+    try
+    {
+        # create the resource group
+        New-AzResourceGroup -Name $resourceGroup -Location $resourceLocation
+
+        # create and check account 1
+        $newTagName = "tag1"
+        $newTagValue = "tagValue1"
+        $retrievedAcc = New-AzNetAppFilesAccount -ResourceGroupName $resourceGroup -Location $resourceLocation -Name $accName1 -Tag @{$newTagName = $newTagValue}
+        Assert-AreEqual $accName1 $retrievedAcc.Name
+        Assert-AreEqual True $retrievedAcc.Tags.ContainsKey($newTagName)
+        Assert-AreEqual "tagValue1" $retrievedAcc.Tags[$newTagName].ToString()
+
+        # create and check account 2 using the Confirm flag
+        $retrievedAcc = New-AzNetAppFilesAccount -ResourceGroupName $resourceGroup -Location $resourceLocation -AccountName $accName2 -Confirm:$false
+        Assert-AreEqual $accName2 $retrievedAcc.Name
+		
+        # create and check account 3 using the WhatIf - it should not be created
+        $retrievedAcc = New-AzNetAppFilesAccount -ResourceGroupName $resourceGroup -Location $resourceLocation -AccountName $accName3 -WhatIf
+
+        # get and check accounts by group (list)
+        $retrievedAcc = Get-AzNetAppFilesAccount -ResourceGroupName $resourceGroup
+        Assert-AreEqual $accName1 $retrievedAcc[0].Name
+        Assert-AreEqual $accName2 $retrievedAcc[1].Name
+        Assert-AreEqual 2 $retrievedAcc.Length
+
+        # get and check an account by name
+        $retrievedAcc = Get-AzNetAppFilesAccount -ResourceGroupName $resourceGroup -Name $accName1
+        Assert-AreEqual $accName1 $retrievedAcc.Name
+
+        # get and check the account again using the resource id just obtained
+        $retrievedAccById = Get-AzNetAppFilesAccount -ResourceId $retrievedAcc.Id
+        Assert-AreEqual $accName1 $retrievedAccById.Name
+
+        # update and check the account (tags)
+        # since only tags can be patched and no other set or update is possible
```

Comment:
Account has actually changed and does require an update function now. This was not previously added because the PowerShell guidelines state that such functions are not required in tag only cases.